include/verilated.mk.in

```diff
@@ -271,10 +271,10 @@ ifneq ($(VM_DEFAULT_RULES),0)
 %.o: %.cpp
 	$(OBJCACHE) $(CXX) $(OPT_FAST) $(CXXFLAGS) $(CPPFLAGS) -c -o $@ $<
 
-$(VK_OBJS_FAST): %.o: %.cpp $(VK_PCH_H).fast.gch
+$(VK_OBJS_FAST): %.o: %.cpp $(VK_PCH_I_FAST)
```

Comment:
VK_PKH_I_FAST includes "-I" and `-I` is not an input file, so this is incorrect.

Reply:
Oops, I screwed up some last second simplification. Fixed.